src/main/java/org/jitsi/impl/reservation/rest/RESTReservations.java

```diff
@@ -283,7 +282,7 @@ private int deleteConference(Number id)
      * @return <tt>Conference</tt> for given <tt>id</tt> or <tt>null</tt>
      *         if not found.
      */
-    private Conference findConferenceForId(Number id)
+    private synchronized Conference findConferenceForId(Number id)
```

Comment:
After adding private to deleteConference, the synchronized here becomes the same: a question of readability.